Kajabi/sage-lib
# [Sage-530] Tabs - Active Indicator Update

## Description

Updates height of active indicator from `4px` to `3px`.

[Figma](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/Sage?node-id=2537%3A21031)

## Screenshots
<img width="354" alt="Screen Shot 2022-05-20 at 11 00 27 AM" src="https://user-images.githubusercontent.com/14791307/169567250-0d314424-e197-4eb0-89be-344bc1cf0f7a.png">


## Testing in `sage-lib`

- View [Tabs](http://localhost:4000/pages/component/tabs?tab=preview)
- Check that active underline is 3px

## Testing in `kajabi-products`

(LOW) Updates height of active indicator from `4px` to `3px`.

## Related

[Sage-530](https://kajabi.atlassian.net/browse/SAGE-530)
